lambda/deploy/example.graphql

```diff
@@ -0,0 +1,12 @@
+schema {
+  query: Query
+}
+
+type User {
+  id: Int
+  name: String
+}
+
+type Query {
+  user: User @http(path: "/users/1", baseURL: "http://jsonplaceholder.typicode.com")
+}
```

Comment:
Do we need this file here?

Reply:
It's only there so we can test the Terraform stuff before `tailcall-on-aws` is made official. I can get rid of it.

Reply:
Resolved in eef1c91cba5171b1dc2a943a5eef5203a3464030